ocaml/bytecomp/bytegen.ml

```diff
@@ -516,6 +516,7 @@ module Storer =
   Switch.Store
     (struct type t = lambda type key = lambda
       let compare_key = Stdlib.compare
+      let join_actions_to_be_shared lam _ = lam
```

Comment:
I think this should be the `Lambda.join_actions_to_be_shared`. It happens to be fine now, since the read semantics are ignored in bytecode, but if that changes I bet no one will think to change this line.